apps/server/swagger/data-api.yml

```diff
@@ -68,7 +118,7 @@
           type: string
         description: Organization name
       - name: entityName
-        description: Array of strings to filter by entity names
+        description: Array of strings to filter by entity names. Ignored if view parameter is `compound`
```

Comment:
same question as before: ignore vs fail, to prevent confusion

Reply:
This comment will be handled in PR[#105](https://github.com/overture-stack/lyric/pull/105)